pkg/devfile/generator/utils.go

```diff
@@ -488,7 +489,7 @@ func getRouteSpec(routeParams RouteSpecParams) *routev1.RouteSpec {
 func getPVCSpec(quantity resource.Quantity) *corev1.PersistentVolumeClaimSpec {
 
 	pvcSpec := &corev1.PersistentVolumeClaimSpec{
-		Resources: corev1.ResourceRequirements{
+		Resources: corev1.VolumeResourceRequirements{
```

Comment:
Not sure if after this change we need to update also the minimum go version here: https://github.com/devfile/integration-tests/blob/f23aefd56e81c69697d546ec19bbffa0f1854e90/README.md?plain=1#L106

Reply:
Good catch, I'll update it.

Reply:
fixed in https://github.com/devfile/integration-tests/pull/105/commits/c9b9207a6939ef3a435c76a80e3cf8f57328a597